playbooks/support/README.md

```diff
@@ -19,111 +50,167 @@ which is determined at least a month in advance (giving people ample time to mak
+participating in ongoing sprint and team activities.
+
+While we would _like_ to have 24/7 on-call coverage, we recognize that we don't yet have a good solution for
+notifying engineers during non-work hours. Do your best to keep an eye out for issues, and lean on those in more
+favorable timezones where appropriate.
+
+## Responding to an incident
+
+### Step 1: Alert
+
+Incidents can be raised by automated or manual means. While on-call, you are responsible for being available to
+respond to these issues.
+
+- Monitor the [#incidents slack channel](https://artsy.slack.com/messages/C9RK0BLEP/) 🔒 for critical issues. This
+  is where urgent issues are raised, either manually or via automated alerts.
+- Monitor the [#alerts](https://artsy.slack.com/messages/C0HP61PUJ/) slack channel _during business/waking hours_.
+  These automated system alerts _may_ be the first sign of a problem but don't _necessarily_ warrant treatment as
+  immediate incidents. E.g., load-balancers may route around failing EC2 instances or auto-scaling may bring up
+  capacity in response to load.
+- If the issue is obviously a non-incident, take the time to educate the reporter on the desired process (see the
+  [examples](#examples) below). You may point them to the
+  [Guide to Reporting Bugs](https://www.notion.so/artsy/Guide-to-reporting-bugs-cc25e1ff41194228b476c4963c646817)
+  doc in Notion which describes this in a general way.
+
+### Step 2: Raise
+
+1. When an incident is raised in the channel or identified in another way, the on-call team first **tracks the
+   incident** by creating a ticket in [Jira Ops](https://artsyproduct.atlassian.net/projects/INCIDENT/incidents),
+   based on the information they know at that time. Some workflow steps:
+
+- In the description on the ticket, link to the slack thread in #incidents.
+- Take your best guess at the other information, it can always be edited later.
+
+2. Using the slack integration on the Jira Ops ticket, create a new slack channel to discuss resolution.
+3. In the original slack thread in #incidents, put a link to the incident that was created and _direct engineers to
+   the newly created slack channel for discussing its resolution_.
+
+### Step 3: Assess and Identify
+
+1. A member of the on-call team marks the issue as **FIXING**. This person will be automatically assigned to the
+   issue. For the duration of the incident resolution, this person is responsible for _facilitation_. This may
+   involve pulling in other engineers to help solve the problem and delegating tasks like external communication to
+   the other on-call team member.
+2. Determine the severity of the incident by trying to reproduce the problem and capturing the answers to the
+   following questions in the Jira Ops ticket:
+
+- How many people (or partners, etc.) does this affect?
+- Have you been able to reproduce?
+- Is this a new problem or something that we are aware of?
```

Comment:
These are helpful for considering severity, but I'm not sure it's necessary to capture the answers to these questions in each incident.

Furthermore this "determine severity" step and the "update severity" step below might be combined.